app/models/task.py

```diff
@@ -1,5 +1,60 @@
-from sqlalchemy.orm import Mapped, mapped_column
+from sqlalchemy.orm import Mapped, mapped_column, relationship
 from ..db import db
+from sqlalchemy import ForeignKey
+from datetime import datetime
+from sqlalchemy import DateTime, Boolean
+from typing import Optional
+from ..models.goal import Goal
+
 
 class Task(db.Model):
     id: Mapped[int] = mapped_column(primary_key=True, autoincrement=True)
+    title: Mapped[str]
+    description: Mapped[str]
+    completed_at: Mapped[Optional[datetime]]
+    is_complete: Mapped[bool] = mapped_column(Boolean, default=False)
+    goal_id: Mapped[Optional[int]] = mapped_column(ForeignKey("goal.id"))
+    goal: Mapped[Optional["Goal"]] = relationship(back_populates="tasks")
```

Comment:
`is_complete` is not intended to be an attribute for `Task` (see Wave 1 documentation [here](https://github.com/Ada-C22/task-list-api/blob/main/ada-project-docs/wave_01.md#task-model) about the 4 attributes the model should start with). `is_complete` should be a value that is calculated by looking at `completed_at` (see Wave 3 docs [here](url) about achieving this).
```suggestion
class Task(db.Model):
    id: Mapped[int] = mapped_column(primary_key=True, autoincrement=True)
    title: Mapped[str]
    description: Mapped[str]
    completed_at: Mapped[Optional[datetime]]
    goal_id: Mapped[Optional[int]] = mapped_column(ForeignKey("goal.id"))
    goal: Mapped[Optional["Goal"]] = relationship(back_populates="tasks")
```